validator/rpc/standard_api.go

```diff
@@ -476,6 +476,39 @@ func (s *Server) SetGasLimit(ctx context.Context, req *ethpbservice.SetGasLimitR
 	return &empty.Empty{}, nil
 }
 
+func (s *Server) DeleteGasLimit(ctx context.Context, req *ethpbservice.DeleteGasLimitRequest) (*empty.Empty, error) {
+	// Responde code when the key is not found on the server.
+	httpCode := "404"
```

Comment:
sorry for this late catch but realized we should be using codes.NotFound for this.
since we use the grpc gateway it converts the gRPC status codes to http status codes, we have to override for 204 but errors can be converted. you can see the conversion here
https://github.com/grpc-ecosystem/grpc-gateway/blob/master/runtime/errors.go

Reply:
Thanks for the reference. Updated the PR.